nifi-nar-bundles/nifi-cdc/nifi-cdc-postgresql-bundle/nifi-cdc-postgresql-processors/.checkstyle

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be required. Could you remove it? Or tell us what it'd be required?

Reply:
The file was removed.

Reply:
:) solved